src/compiler/file_uploader/on_before_exit.ts

```diff
@@ -46,6 +27,6 @@ async function cleanup(
 ): Promise<void> {
     const incompleteFiles = await getListOfIncompleteFiles(paths, actor);
     for (const [_, path] of incompleteFiles) {
-        await actor.clear_file_and_info(path);
+        await actor._azle_clear_file_and_info(path);
```

Comment:
Can this not be removed? Done in the same update calls as the chunking?

Reply:
We're in the middle of accomplishing this

Reply:
https://github.com/demergent-labs/azle/issues/1995

Reply:
The issue linked in the previous reply doesn't seem correct